README.md

```diff
@@ -7,36 +7,52 @@ Use this repository to submit official Charts for NuoDB. Charts are curated appl
 
 For more information on using Helm, refer to the [Helm's documentation](https://github.com/kubernetes/helm#docs).
 
-## How to use this repository?
+## Software Release requirements
 
-For a list of supported NuoDB Helm Chart releases and where to download, click the `Releases` tab above.
+| Software   | Release Requirements                           | 
+|------------|------------------------------------------------|
+| Kubernetes |  The latest and previous minor versions of Kubernetes. For example, if the latest minor release of Kubernetes is 1.15 then 1.15 and 1.14 are supported. Charts may still work on previous versions of Kubernertes even though they are outside the target support window.|
+| Helm       |  Version 2, 2.9 or greater   |
```

Comment:
But not 3.0...

Reply:
Why are we mentioning 2 and 2.9? What about 2.1-2.8?

We should just say 2.9 or greater.

Reply:
How about:
`Version 2.9 and greater - and less than version 3.0`

Reply:
Martin, 
We cannot say just "2.9 or greater" because we are left in the situation. Unclear about version 3.  As Nik has indicated, and I tried as well in my version, to show that v3 is not supported. I'll try to revise it further.

Reply:
thanks for explaining that. I completely forgot that Helm 3 is a rewrite.